troposphere/static/js/components/admin/AtmosphereUser.jsx

```diff
@@ -60,7 +61,7 @@ export default React.createClass({
             <td style={{ border: "none" }} 
                 className="user-name"
             >
-                {user.get("username")}
+                {user.get("username")} <Emulate username={user.get("username")} />
```

Comment:
It would be helpful to offer some reasoning because without it seems to be a personal preference. Reasons like "we want to ensure that at least 5 em exists between elements". I know spacing & alignment are often the "little" things that impact a design. 

Regarding _mobile-friendliness_,  the "username" text isn't clickable, so the current placement wouldn't create frustration with "target" a clickable element.